targets/targets.json

```diff
@@ -4323,6 +4323,42 @@
             "network-default-interface-type": "ETHERNET"
         }
     },
+    "MIMXRT1170_EVK": {
+        "supported_form_factors": [
+            "ARDUINO"
+        ],
+        "core": "Cortex-M7FD",
+        "supported_toolchains": [
+            "GCC_ARM"
+        ],
+        "extra_labels": [
+            "NXP",
+            "MCUXpresso_MCUS",
+            "EVK",
+            "MIMXRT1170",
+	    "IMX"
+        ],
+        "is_disk_virtual": true,
+        "macros": [
+            "CPU_MIMXRT1176DVMAA_cm7",
+            "XIP_BOOT_HEADER_ENABLE=1",
+            "XIP_EXTERNAL_FLASH=1",
+            "XIP_BOOT_HEADER_DCD_ENABLE=1",
+            "SKIP_SYSCLK_INIT",
+            "MBED_MPU_CUSTOM"
+        ],
+        "inherits": [
+            "Target"
+        ],
+        "detect_code": [
+            "0228"
+        ],
+        "device_has": [
+            "SERIAL",
+	        "LPTICKER"
+        ],
+        "device_name": "MIMXRT1170"
```

Comment:
travis reports an error with

`AssertionError: Target MIMXRT1170_EVK contains invalid device_name MIMXRT1170`. You can either update the index.json from the cmsis pack (https://os.mbed.com/docs/mbed-os/v6.8/program-setup/adding-and-configuring-targets.html) for this device or remove this

Reply:
ok fixed it, is there a way to run this travis test? prior to pull request so I can check all of this

Reply:
Each stage in Travis matrix use a script or cmd, I usually run the same command locally to reproduce an issue. Often just going through the logs and finding an offender is sufficient.

Reply:
ok sounds good ill run the commands locally to double check myself before. thank you